crypto/x509/x509_vfy.c

```diff
@@ -326,26 +340,10 @@ static X509 *find_issuer(X509_STORE_CTX *ctx, STACK_OF(X509) *sk, X509 *x)
     return rv;
 }
 
-/*
- * Check that the given certificate 'x' is issued by the certificate 'issuer'
- * and the issuer is not yet in ctx->chain, where the exceptional case
- * that 'x' is self-issued and ctx->chain has just one element is allowed.
- */
+/* Check that the given certificate 'x' is issued by the certificate 'issuer' */
 static int check_issued(X509_STORE_CTX *ctx, X509 *x, X509 *issuer)
```

Comment:
As ctx is unused and the function is truly rudimentary, perhaps drop it altogether?

Reply:
I am sorry, too little coffee this morning. It of course has to stay with this signature. Then perhaps mark the ctx as ossl_unused?

Reply:
I also noticed that the function has become pretty simple - and thus much easier to understand :smile:
but we cannot drop it because it is needed by `X509_STORE_CTX_init()` as default implementation:
```
       ctx->check_issued = check_issued;
```

Reply:
Ah, just saw your second thought. Will do.